docs/src/components/TailwindClassnames/index.tsx

```diff
@@ -52,6 +69,49 @@ const getCategory = (name: string) => {
   return "palette";
 };
 
+const transformColors = (colors: Record<string, string>) => {
+  return Object.entries(colors).reduce((acc, [category, value]) => {
+    if (typeof value === "object") {
+      Object.entries(value).forEach(([name, color]) => {
+        acc[`${category}-${name}`] = color;
+      });
+    } else {
+      acc[category] = value;
+    }
+
+    return acc;
+  }, {});
+};
+
+const CategoryDescriptions = ({ category }: { category: Category }) => {
+  return (
+    <>
+      {category === "Colors" && (
+        <Text type="info" weight="medium" margin={{ bottom: "12px" }}>
+          To be used with Tailwind classes that apply a color property (eg: `text-blue-dark`,
+          `bg-red-light`, etc).
+        </Text>
+      )}
+      {category === "Screens" && (
+        <Text type="info" weight="medium" margin={{ bottom: "12px" }}>
+          To be used as a Tailwind prefix for defining media queries (eg: `sm:opacity-1`,
+          `tb:hidden`, etc).
+        </Text>
+      )}
+      {category === "Spacing" && (
+        <Text type="info" weight="medium" margin={{ bottom: "12px" }}>
+          To be used as a Tailwind prefix for defining margin and padding (eg: `m-1`, `p-2`, etc).
+        </Text>
+      )}
+      {category === "ZIndex" && (
+        <Text type="info" margin={{ bottom: "12px" }}>
```

Comment:
All others have `weight="medium"` except this one. Which leads me to think…
Couldn't we have something like

```
 <Text type="info" weight="medium" margin={{ bottom: "12px" }}>
  {category === ...}
</Text>
```

So that we only write the `<Text>` code once and the logic of the string is calculated inside? Or maybe abstract it before the return… This way we can avoid the fragment at all, no?